.github/workflows/cypress.yml

```diff
@@ -39,4 +39,4 @@ jobs:
           wait-on-timeout: 30
           quiet: true
           browser: chrome
-          spec: cypress/integration/parallel-${{ matrix.group }}/*.ts
+          spec: cypress/e2e/parallel-${{ matrix.group }}/*.ts
```

Comment:
Why do we need to rename this path?

Reply:
the migration renamed the folder, I guess because Cypress started referring to these type of tests as end to end rather than integration